### PR TITLE
Recurring buy 2

### PR DESCRIPTION
**Pull request by prwelber**

### CODE REVIEW

**Comment by prwelber** [comment]
This adds a check to show/hide recurring buy based on `Env`, so we'll have to add it to wallet options as well when this goes to staging and beyond.

File: assets/js/components/exchange/exchange-recurring.component.js [hunk 1]
```diff
@@ -18,6 +18,8 @@ function ExchangeRecurringController ($scope, currency, recurringTrade) {
   $scope.date = new Date();
   $scope.format = currency.formatCurrencyForView;
 
-  $scope.endTime = this.endTime && this.endTime.toDateString();
+  this.trade.fiatAmount = !this.trade.fiatAmount ? this.trade.inAmount / 100 : this.trade.fiatAmount;
```
Comment (plondon) [inline assets/js/components/exchange/exchange-recurring.component.js]: this.trade.fiatAmount || this.trade.inAmount / 100

**Comment by plondon** [comment]
uib-datepicker could use some more love. Good idea to add it to bs-overrides

screen shot 2017-10-26 at 1 21 54 pm

**Comment by plondon** [comment]
Looks really nice 👏

**Pull request merged**